### PR TITLE
DISPLAY-929: Added thumbnails for images

### PR DESCRIPTION
**Pull request by tuj**
#### Link to ticket

https://jira.itkdev.dk/browse/DISPLAY-929

#### Description

Added thumbnails for images

#### Checklist

- [ ] My code is covered by test cases.
- [ ] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

### CODE REVIEW

**Review by turegjorup — CHANGES_REQUESTED** [review]
Nice to have: Cleanup thumbnail on media delete

File: composer.json [hunk 1]
```diff
@@ -122,6 +123,11 @@
         "fixtures-load": [
             "bin/console hautelook:fixtures:load --no-interaction"
         ],
+        "prepare-code": [
+            "@coding-standards-apply",
+            "@tests",
+            "@update-api-spec"
```
[the remark this answers is not in the document]
Comment (tuj) [reply composer.json]: Have removed it for now. I generally think tools should make things as easy as possible. What is committed is the developers responsibility.

File: config/packages/liip_imagine.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Documentation on how to configure the bundle can be found at: https://symfony.com/doc/current/bundles/LiipImagineBundle/basic-usage.html
+liip_imagine:
+    # valid drivers options include "gd" or "gmagick" or "imagick"
+    driver: "gd"
```
[the remark this answers is not in the document]
Comment (tuj) [reply config/packages/liip_imagine.yaml:4]: Added

File: composer.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (turegjorup) [reply composer.json]: But then we should just remove the API spec check from actions....

Having a check that makes it more difficult to commit something in error, and then updating our tools to make it easier to bypass that check is just stupid.

**Review by turegjorup — APPROVED** [review]

**Pull request merged**